storage/interface.go

```diff
@@ -56,6 +56,13 @@ type MetricStore interface {
 	// undefinded state). If nil is returned, the MetricStore cannot be
 	// "restarted" again, but it can still be used for read operations.
 	Shutdown() error
+	// Returns true, nil if the MetricStore is currently working as expected
+	// or false, Error if it is not.
+	Healthy() (bool, error)
```

Comment:
Couldn't we just return `error` with the usual sentinel semantics that `nil` is OK and everything else is an error?

Reply:
good idea